cmd/cel-key/main.go

```diff
@@ -12,11 +12,11 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/spf13/cobra"
 
-	"github.com/celestiaorg/celestia-app/v3/app"
-	"github.com/celestiaorg/celestia-app/v3/app/encoding"
+	"github.com/celestiaorg/celestia-app/v4/app"
+	"github.com/celestiaorg/celestia-app/v4/app/encoding"
 )
 
-var encodingConfig = encoding.MakeConfig(app.ModuleEncodingRegisters...)
+var encodingConfig = encoding.MakeTestConfig(app.ModuleEncodingRegisters...)
```

Comment:
this might look unusual because of the naming. In `celestia-app` the `TestConfig` is an encoding config that has all types registered (what I assume is desired here) while `MakeConfig` constructs a config that is later updated with the different types based on the sdk app module life cycle functions.

Reply:
You will have to update this to `encoding.MakeConfig(...)` when you update to latest of v4. I PR'd a refactor that managed to get rid of this separate test helper yesterday

Reply:
yeah I saw that! Should just be a one liner right?